### PR TITLE
Clean README

### PR DESCRIPTION
**Pull request by xuanquynh**
I imagine that I am a new one who wants to use this package and try to make the documentation nicer.

I also help to fix some typo such as "of you required composer packages".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,59 +6,56 @@ Show unused composer dependencies by scanning your code
...
-Working in a big repository with multiple people, sometimes you might lose track 
-of you required composer packages. You have so many packages you can't be sure if they are used
-or not.
+When working in a big repository, you sometimes lose track of your required composer
+packages. There may be so many packages you can't be sure if they are used or not.
 
-You can use `composer why package/A` but that only gives you information about why this package is installed
-in dependency to another package.
+Unfortunately, the `composer why` command only gives you the information about why
+a package is installed in dependency to an another package.
```
Comment (icanhazstring) [inline README.md]: "to another package"  the `an` is too much here

File: README.md [hunk 2]
```diff
@@ -6,59 +6,56 @@ Show unused composer dependencies by scanning your code
...
 
-What if you need to find out if the provided *namespaces* of the package are used in your code?
+How do we check whether the provided *namespaces* of a package are used in our code?
 
-`composer unused` to the rescue!
+`composer unused` is created to the rescue!
```
Comment (icanhazstring) [inline README.md]: "`composer unused` to the rescue" was correct. No change needed here.

**Comment by icanhazstring** [comment]
Thanks for the improvements. Some changes needed tho :)

**Comment by xuanquynh** [comment]
I resolved your comments.

**Comment by icanhazstring** [comment]
Thank you. Merged.

**Pull request merged**